.env

```diff
@@ -1 +1,2 @@
 TAG=dev
+HTTP_PORT=2000
```

Comment:
This is great! But can we please leave the default HTTP port to 80?

I think it should be possible to set a different default in the docker-compose.dev.yml overlay if you want to have a dev-time port other than 80. However, I'd like for that default to be 5000, since this has been the historical default for the dev tooling.